main.rb

```diff
@@ -0,0 +1,47 @@
+require './app'
+
+def show_menu
+  puts "\nPlease choose an option by entering a number:
+1 - List all books
+2 - List all people
+3 - Create a person
+4 - Create a book
+5 - Create a rental
+6 - List all rentals for a given person id
+7 - Exit"
+end
+
+def process_options(usr_inp)
+  case usr_inp
+  when '1'
+    @app.list_all_books
+  when '2'
+    @app.list_all_people
+  when '3'
+    @app.create_person
+  when '4'
+    @app.create_book
+  when '5'
+    @app.create_rental
+  when '6'
+    @app.list_rentals
+  else
+    exit
+  end
+end
+
+def main
+  @app = App.new
+  $stdout.clear_screen
```

Comment:
- [ ] Good job so far, when I comment this line `$stdout.clear_screen`  and the code used to test class decorated in [person.rb](https://github.com/lucy-sees/Libra/blob/70995b8a64bc6c2694b201dd81b2e40cb119caaf/person.rb#L39) I can see the options presented to the user, Kindly make sure that your app works as expected  without errors

Reply:
I have already made this change

Reply:
- [ ] Hi @lucy-sees, as you see the part I highlighted  in this review was green, I only saw your updates after I submitted my review 👍🏿